src/lib/room.js

```diff
@@ -162,15 +162,38 @@ export const loadMessages = async () => {
 	}
 
 	await store.setState({ loading: true });
-	const rawMessages = (await Livechat.loadMessages(rid)).concat(previousMessages);
-	const messages = (await normalizeMessages(rawMessages)).map(transformAgentInformationOnMessage);
-
-	await initRoom();
-	await store.setState({ messages: (messages || []).reverse(), noMoreMessages: false, loading: false });
-
-	if (messages && messages.length) {
-		const lastMessage = messages[messages.length - 1];
-		await store.setState({ lastReadMessageId: lastMessage && lastMessage._id });
+	try {
+		const rawMessages = (await Livechat.loadMessages(rid)).concat(previousMessages);
+		const messages = (await normalizeMessages(rawMessages)).map(transformAgentInformationOnMessage);
+		await initRoom();
+		await store.setState({ messages: (messages || []).reverse(), noMoreMessages: false, loading: false });
+
+		if (messages && messages.length) {
+			const lastMessage = messages[messages.length - 1];
+			await store.setState({ lastReadMessageId: lastMessage && lastMessage._id });
+		}
+	}
+	catch(e) {
+		const waitTime = parseInt(e.data.error.match(/^.*You must wait ([0-9]*) seconds.*$/)[1]);
+		const decrementCounter = async () => {
+			if(window.loadMessagesTimeout) {
+				clearInterval(window.loadMessagesTimeout);
+			}
+			window.loadMessagesTimeout = setInterval(() => {
+				let tmpWaitTime = store.state.loadWaitTime;
+				if(tmpWaitTime) {
+					store.setState({loadWaitTime: tmpWaitTime - 1, loadingMessage: `Error, too many requests. Please slow down. You must wait for ${tmpWaitTime} seconds.` });
```

Comment:
Please set this message on i18n files.

Reply:
according to your comment, does I need to set that message in all i18n files. Like I have to translate that message to all other languages and set it to respective i18n json files. Right? And if this is the case how I can identify which i18n json file belong to which language. Is there any fast way to do this or I have to manually check each i18n json file to check which language it belongs to. Looking forward to your response. Thanks.

Reply:
please check the `i18n` command on `package.json` files.

You'll use with `I18n.t('YOUR TEXT')` and run the command, that command will create the line on i18n files.